addons/osio-addon/pom.xml

```diff
@@ -32,6 +32,10 @@
       <artifactId>launcher-service-openshift-impl</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.wildfly.swarm</groupId>
+      <artifactId>jaxrs-validator</artifactId>
```

Comment:
I don't think this dependency is necessary, have you added for the test?

Reply:
fixed